<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="modelChoice" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>491</height>
    </rect>
   </property>
   <property name="tabPosition" >
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="determinist" >
    <attribute name="title" >
     <string>Modèle déterministe</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_filename" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="editable" >
      <bool>false</bool>
     </property>
     <item>
      <property name="text" >
       <string>Choisir répertoire...</string>
      </property>
     </item>
    </widget>
    <widget class="QTabWidget" name="tabWidget_algo" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>210</y>
       <width>558</width>
       <height>91</height>
      </rect>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>VNS</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_nbIterationsVNS" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>10</y>
         <width>110</width>
         <height>27</height>
        </rect>
       </property>
       <property name="inputMask" >
        <string>D999999999; </string>
       </property>
       <property name="text" >
        <string>3</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>132</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Nombre d'itérations :</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Recuit simulé</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>0</y>
         <width>298</width>
         <height>66</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Température initiale :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_initialTemp" >
            <property name="inputMask" >
             <string>d999999999; </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Nombre d'itérations :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_nbIterationsRecuit" >
            <property name="inputMask" >
             <string>d999999999; </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Relaxation Lagrangienne</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget" >
       <property name="geometry" >
        <rect>
         <x>11</x>
         <y>10</y>
         <width>389</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Numéro de la (ou les) contrainte(s) à relacher:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_constraintToRelax" />
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>225</x>
       <y>0</y>
       <width>445</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Nombre n de titres disponibles :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nbAvalableTitles" >
          <property name="text" >
           <string>0</string>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Espérance désirée du rendement :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_esperance" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Nombre K de titres à retenir pour constituer le portefeuille :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_K" >
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>350</y>
       <width>198</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Proportions des titres retenus :</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_proportionsMinMax" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>100</y>
       <width>741</width>
       <height>111</height>
      </rect>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <row>
      <property name="text" >
       <string>Proportion minimale</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>Proportion maximale</string>
      </property>
     </row>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QTableWidget" name="tableWidget_resultProportions" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>370</y>
       <width>741</width>
       <height>75</height>
      </rect>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
     <property name="gridStyle" >
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled" >
      <bool>false</bool>
     </property>
     <row>
      <property name="text" >
       <string>Proportion</string>
      </property>
     </row>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="label_resultat" >
     <property name="geometry" >
      <rect>
       <x>164</x>
       <y>310</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>127</red>
           <green>125</green>
           <blue>123</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string> </string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget" >
     <property name="geometry" >
      <rect>
       <x>584</x>
       <y>210</y>
       <width>167</width>
       <height>96</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Temps maximum de calcul:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_maxComputation" >
        <property name="time" >
         <time>
          <hour>5</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_run" >
        <property name="font" >
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Run!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="stochastic" >
    <attribute name="title" >
     <string>Modèle stochastique</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2" >
     <property name="geometry" >
      <rect>
       <x>26</x>
       <y>220</y>
       <width>558</width>
       <height>91</height>
      </rect>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>VNS</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2" >
       <property name="geometry" >
        <rect>
         <x>21</x>
         <y>10</y>
         <width>294</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Nombre d'itérations :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_2" >
          <property name="value" >
           <number>50</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2" >
          <property name="value" stdset="0" >
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5" >
      <attribute name="title" >
       <string>Recuit simulé</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_3" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>0</y>
         <width>298</width>
         <height>66</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Température initiale :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>Nombre d'itérations :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_6" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6" >
      <attribute name="title" >
       <string>Relaxation Lagrangienne</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_4" >
       <property name="geometry" >
        <rect>
         <x>11</x>
         <y>10</y>
         <width>389</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_14" >
          <property name="text" >
           <string>Numéro de la (ou les) contrainte(s) à relacher:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_7" />
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_5" >
     <property name="geometry" >
      <rect>
       <x>590</x>
       <y>220</y>
       <width>167</width>
       <height>96</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Temps maximum de calcul:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit_2" />
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="font" >
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Run!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
